src/Symfony/Bridge/Twig/Extension/WorkflowExtension.php

```diff
@@ -117,7 +117,7 @@ public function getMarkedPlaces($subject, $placesNameOnly = true, $name = null)
      *                                                Use a string (the place name) to get place metadata
      *                                                Use a Transition instance to get transition metadata
      */
-    public function getMetadata($subject, string $key, $metadataSubject = null, string $name = null): ?string
+    public function getMetadata($subject, string $key, $metadataSubject = null, string $name = null)
```

Comment:
what about adding `@return mixed type declared inside your workflow configuration`

Reply:
It does not bring much value IMHO